### PR TITLE
jackett indexers as of 3391d0f

### PR DESCRIPTION
**Pull request by bakerboy448**
jackett indexers as of 3391d0f
jackett indexers as of 2b227ff
jackett indexers as of 68d67f5

### CODE REVIEW

File: definitions/v2/mircrew.yml [hunk 1]
```diff
@@ -40,6 +40,7 @@ caps:
     - {id: 53, cat: Movies, desc: "Richieste Film"}
     - {id: 54, cat: TV, desc: "Richieste Serie TV"}
     - {id: 55, cat: Movies, desc: "Richieste Video diverse"}
+    - {id: 57, cat: Movies, desc: "Richieste Video Soddisfatte"}
```
Comment (Qstick) [inline definitions/v2/mircrew.yml:43]: Should this be on v1 also?

**Review by Qstick — CHANGES_REQUESTED** [review]

File: definitions/v2/mircrew.yml (hunk 1 above)
Comment (bakerboy448) [reply definitions/v2/mircrew.yml:43]: No. They are not a v1 indexer.

utilizing this indexer in Prowlarr is blocked by the Selctors / V2 PR

**Comment by bakerboy448** [comment]
updated through 2b227ff
recommend holding until jackett cuts another release

**Comment by bakerboy448** [comment]
seems garfield is a on a roll

lets push this now and then push with the next release later this weekend

File: definitions/v2/elitetorrent-biz.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ language: es-es
 type: public
 encoding: UTF-8
 links:
-  - https://elitetorrent.si/
+  - https://elitetorrent.com/
```
Comment (Qstick) [inline definitions/v2/elitetorrent-biz.yml:9]: Should this also be in v1

File: definitions/v2/mircrew.yml (hunk 1 above)
Comment (Qstick) [reply definitions/v2/mircrew.yml:43]: We didn't backport them? It's a few lines right

**Review by Qstick — CHANGES_REQUESTED** [review]

File: definitions/v2/elitetorrent-biz.yml (hunk 2 above)
Comment (bakerboy448) [reply definitions/v2/elitetorrent-biz.yml:9]: They don't have a v1

File: definitions/v2/mircrew.yml (hunk 1 above)
Comment (bakerboy448) [reply definitions/v2/mircrew.yml:43]: It'd be needing to rewrite the download implementation To not use selectors.

that applies to any of the only v2 indexers.

File: definitions/v2/elitetorrent-biz.yml (hunk 2 above)
Comment (Qstick) [reply definitions/v2/elitetorrent-biz.yml:9]: Sure?

File: definitions/v2/mircrew.yml (hunk 1 above)
Comment (Qstick) [reply definitions/v2/mircrew.yml:43]: Yea seems they use the selectors to hack the link on this one, so good. I made v1 for elite, so just that update needed.

File: definitions/v2/elitetorrent-biz.yml (hunk 2 above)
Comment (bakerboy448) [reply definitions/v2/elitetorrent-biz.yml:9]: Touché 
Pushing momentarily

**Review by Qstick — APPROVED** [review]

**Pull request merged**